src/main/java/org/example/Initializer.java

```diff
@@ -22,7 +19,28 @@ public class Inicializer {
     QuestionService questionService = new QuestionService(inputDialog, outputDialog);
     PriceService priceService = new PriceService(questionService);
     PolicyCreator policyCreator = new PolicyCreator(inputDialog,priceService,questionService);
+    PolicyService policyService = new PolicyService(inputDialog, outputDialog, repositoryPolicy, policyCreator);
 
-    public Inicializer() throws IOException {
+    public Initializer() throws IOException {
```

Comment:
This is a good start.
We should only try to make things public that are used within the production code itself (and not just used for set up).

For example, getRepositoryPolicy() is not used anywhere in the codebase and therefore this method should not exist.
LoginService is being used and should have a method called getLoginService.

What I would recommend doing to identify this is to make **all** the fields in this class private (for example `private final RepositoryLogIn repositoryLogIn = new RepositoryLogIn(connection);`) and see where the code complains.

You should only create getter method (e.g. getPolicyService) for things that are being used outside of this Initializer.

Let me know if that makes sense 😄